content/en/docs/cluster-administration/storageclass.md

```diff
@@ -1,8 +1,41 @@
 ---
 title: "StorageClass"
-keywords: "kubernetes, docker, kubesphere, jenkins, istio, prometheus"
-description: "Kubernetes and KubeSphere node management"
+keywords: "kubernetes, docker, kubesphere, storage class, Ceph RBD, Glusterfs, QingCloud, "
+description: "KubeSphere StorageClass management"
 
 linkTitle: "StorageClass"
 weight: 100
 ---
+## Introduction
+StorageClasses are used to provide storage resources, always PersistentVolume. 
+On KubeSphere, `Ceph RBD` `Glusterfs` `QingCloud-CSI` can be created on console.
+If other storage to be used, you can create a custom StorageClass.
+
+## Create StorageClasses
+
+![Create Storage Class](/images/storage/create-storage-class.png)
+
+### Ceph RBD
+Ceph RBD is an in-tree storage plugin on Kubernetes. 
+StorageClass parameters could be seen in [Kubernetes Docs](https://kubernetes.io/docs/concepts/storage/storage-classes/#ceph-rbd).
+
+- Storage System: As **hyperkube** images were [deprecated since 1.17](https://github.com/kubernetes/kubernetes/pull/85094), in-tree Ceph RBD may not work without **hyperkube**.
+Fortunately, [rbd provisioner](https://github.com/kubernetes-incubator/external-storage/tree/master/ceph/rbd) could be used as a substitute, 
+which format is the same as in-tree Ceph RBD. The only different parameter is the **provisioner** here is the **storage system**. 
+So if you has installed and choose to use rbd provisioner, the parameter **Storage System** should be **ceph.com/rbd**.
+If you use in-tree Ceph RBD, the parameter **storage system** should be **kubernetes.io/rbd**. 
+
+### Glusterfs
+Glusterfs is also an in-tree storage plugin on Kubernetes. 
+StorageClass parameters could be seen in [Kubernetes Docs](https://kubernetes.io/docs/concepts/storage/storage-classes/#glusterfs).
+
+### QingCloud CSI
+QingCloud CSI is the storage plugin on Kubernetes for the disk of QingCloud. 
+Before creating StorageClass, QingCloud CSI plugin should be installed on the cluster. 
```

Comment:
->
Before you create the StorageClass, QingCloud CSI plugin should be installed on the cluster.